synfig-core/src/synfig/CMakeLists.txt

```diff
@@ -145,7 +145,7 @@ install(
 install(
     TARGETS synfig
     EXPORT synfig
-    LIBRARY DESTINATION lib
+    LIBRARY DESTINATION ${CMAKE_BINARY_DIR}/output/lib
```

Comment:
here need to be relative path

Reply:
But `2-build-debug.sh` builds it inside a `build` folder. 
```
  - _debug
    - build
      - lib
        - libsynfig.so  
```

Making structure like:
```
  - cmake-build-debug
    - output
      - lib
        - libsynfig.so
```

Reply:
Ah, okay, nevermind. It works the way you told too.